frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -17,6 +17,7 @@ export const podColor = {
 };
 
 export const podStatus = Object.keys(podColor);
+const IS_KNATIVE = 'serving.knative.dev/service';
```

Comment:
We should move this constant to the knative plugin package and rename it something like KNATIVE_SERVING_LABEL